scripts/complete_liquidity_provision.js

```diff
@@ -92,11 +94,18 @@ module.exports = async callback => {
         callback("Error: Bound checks did not pass")
       }
     }
+    if (argv.fleetSize > 23) {
+      callback("Error: Choose a smaller fleetSize, otherwise your payload will be to big for Metamask")
```

Comment:
I tested it and this limit is enforced by Infura, if you use a different node the transaction goes through smoothly.